config.py

```diff
@@ -410,6 +410,72 @@
             '.*[0-9]+.*',
         ],
     ),
+
+    nb = dict(
+        lang_name = 'Norsk (bokmål)',
+        lang_dir = 'ltr',
+        database = 'nowiki_p',
+        wikipedia_domain = 'no.wikipedia.org',
+
+        beginners_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder',
+        beginners_link_title = 'Wikipedia:Bruk av kilder',
+        lead_section_policy_link = 'https://no.wikipedia.org/wiki/Wikipedia:Bruk_av_kilder#Hvorfor_siterer_vi_kilder',
+        lead_section_policy_link_title = 'Wikipedia:Bruk_av_kilder',
+
+        citation_needed_category = 'Artikler som trenger referanser',
+        citation_needed_templates = EN_CITATION_NEEDED_TEMPLATES + [
+            'Trenger referanse',
+            'Tr',
+            'Referanse',
+        ],
+        citation_needed_template_name = 'trenger referanse',
+        wikilink_prefix_blacklist = EN_WIKILINK_PREFIX_BLACKLIST,
+        tags_blacklist = EN_TAGS_BLACKLIST,
+        templates_blacklist = EN_TEMPLATES_BLACKLIST,
+        hidden_category = 'Skjulte kategorier',
+        category_name_regexps_blacklist = [
+            '.*[0-9]+.*',
+        ],
+    ),
+
+    nn = dict(
```

Comment:
579 doesn't sound wrong actually, it's a pretty small wiki. They have ~2000 articles in the category for missing references, but most of those are via a general template saying that the _entire article_ is missing references.